index.html

```diff
@@ -12,14 +12,14 @@
   <body>
     <header class="toolbar">
       <div class="myLogo">My Logo</div>
-      <div class="nav">
+      <nav class="nav-bar">
         <img
           src="./assets/Enabled.svg"
           alt="hamburger icon"
           width="40"
           height="40"
         />
```

Comment:
- [ ] [OPTIONAL] The hamburger button has a `<nav>` tag and I think it would be nice if we followed this structure 👇🏻
```
<header>
  <a href=""> -- logo goes here -- </a>
  <nav>
    <button> -- menu image goes here -- </button>
  </nav>
  <ul>
    <li><a href="#">Home</a></li>
    <li><a href="#">About</a></li>
    <li><a href="#">Contact</a></li>
  </ul>
</header>
```
❓ What do you think about this structure?

Reply:
You are right, It looks nicer, cleaner, more descriptive. I will change it.

Reply:
I think you should eliminate the `<ul>` part because its not needed for this milestone

Reply:
Ok, thank you. Anyway, I wrapped the hamburger icon within a button tag because it needs to be clickable.